nautobot_floor_plan/tables.py

```diff
@@ -43,3 +43,50 @@ class Meta(BaseTable.Meta):
             "tags",
             "actions",
         )
+
+
+class FloorPlanTileTable(BaseTable):
+    # pylint: disable=R0903
+    """Table for list view."""
+
+    floor_plan_tile = tables.Column(empty_values=[])
+    floor_plan = tables.Column(empty_values=[])
+    rack = tables.Column(linkify=True)
+    tags = TagColumn()
+    actions = ButtonsColumn(models.FloorPlanTile)
+
+    def render_floor_plan_tile(self, record):
+        """Render a link to the detail view for the FloorPlanTile record itself."""
+        return hyperlinked_object(record)
+
+    def render_floor_plan(self, record):
+        """Render a link to the detail view for the FloorPlan record itself."""
+        return hyperlinked_object(record.floor_plan)
```

Comment:
Surprised these are needed - does it "just work" if you change the column definitions for these two to add `linkify=True` to them?

Reply:
@glennmatthews 

I discovered I do not need the `render_floor_plan` method. However, I cannot figure out how to render the model's `__str__` method any other way.  I discovered the `accessor` param but I am not sure there's an alternative outside the `render_x_()` approach?  This is where it stands - 

```python
class FloorPlanTileTable(BaseTable):
    # pylint: disable=too-few-public-methods
    """Table for list view."""

    floor_plan_tile = tables.Column(verbose_name="Tile",empty_values=[])
    floor_plan = tables.Column(linkify=True)
    location = tables.Column(accessor="floor_plan__location",linkify=True)
    rack = tables.Column(linkify=True)
    tags = TagColumn()
    actions = ButtonsColumn(models.FloorPlanTile)

    def render_floor_plan_tile(self, record):
        """Render a link to the detail view for the FloorPlanTile record itself."""
        return hyperlinked_object(record)
        
    class Meta(BaseTable.Meta):
        """Meta attributes."""
        
        --ommitted--
```

Reply:
That's probably fine as-is then. Thanks for investigating!